tweetapi/login.py

```diff
@@ -71,6 +70,7 @@ def get(self,request):
         return Response(serializer.data)
 
 class Logout(APIView):
+    authentication_classes=[Authentication]
```

Comment:
this should be default in settings

Reply:
when on default login it does not allow login

Reply:
then you set authentication_classes to [] for login

Reply:
thank you sir, i didnt know i could do this.